src/app/board/question/hot/page.tsx

```
@@ -0,0 +1,136 @@
+
+  // 데이터 로드 함수
+  const fetchHotQuestions = async (tabType: "주간" | "일간" = "주간") => {
+    setIsLoading(true);
+    try {
+      const response =
+        tabType === "주간"
+          ? await questionPostApi.getWeeklyPopularQuestionPost()
+          : await questionPostApi.getDailyPopularQuestionPost();
+
+      if (response && response.questionPostsPage) {
+        setQuestionData(response.questionPostsPage.content || []);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)